code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -106,7 +106,7 @@
 /obj/structure/closet/secure_closet/chemical
 	name = "chemical closet"
 	desc = "Store dangerous chemicals in here."
```

Comment:
seems like we're moving away from this being the purpose of the locker

Reply:
I mean, that's just due to the fact it's filled with misc items rather than what's advertised